### PR TITLE
ISC - update Fee Policy

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

* removed outdated FeePolicy info
* added explanation on the gas policy in EVM compatibility.mdx
* added admonition explaining where to query for the current gas

## Type of change

- Documentation Fix

## Change checklist

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: docs/build/isc/v1.1/docs/_admonitions/_query_gas_fees.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+:::tip Current Gas Fee
+
+You can get the current average gas fee by querying the `eth_gasPrice` using the [JSON-RPC API](../reference/json-rpc-spec.md/#json-rpc-methods-according-to-ethereum-client-api).
```
Comment (jorgemmsilva) [inline docs/build/isc/v1.1/docs/_admonitions/_query_gas_fees.md:3]: this is wrong. "minimum", not average.

File: docs/build/isc/v1.3-alpha/docs/_admonitions/_query_gas_fees.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+:::tip Current Gas Fee
+
+You can get the current average gas fee by querying the `eth_gasPrice` using the [JSON-RPC API](../reference/json-rpc-spec.md/#json-rpc-methods-according-to-ethereum-client-api).
```
Comment (jorgemmsilva) [inline docs/build/isc/v1.3-alpha/docs/_admonitions/_query_gas_fees.md:3]: wrong. why is this text duplicated?

File: docs/build/isc/v1.3-alpha/docs/getting-started/compatibility.mdx [hunk 3]
```diff
@@ -43,4 +44,9 @@ functionalities, introducing commands like `isc.send(...)` for token transfers.
 
 ### Gas Fees
 
-Gas fees are set by the chain.
+As in [Ethereum](https://ethereum.org/en/developers/docs/gas), gas fees depend on the current network usage. As in an
```
Comment (jorgemmsilva) [inline docs/build/isc/v1.3-alpha/docs/getting-started/compatibility.mdx:47]: duplicated?

File: docs/build/isc/v1.3-alpha/docs/_admonitions/_query_gas_fees.md (hunk 2 above)
Comment (lucas-tortora) [reply docs/build/isc/v1.3-alpha/docs/_admonitions/_query_gas_fees.md:3]:
```suggestion
You can get the minimum gas fee by querying the `eth_gasPrice` using the [JSON-RPC API](../reference/json-rpc-spec.md/#json-rpc-methods-according-to-ethereum-client-api).
```

**Pull request merged**